pkg/controller/controller.go

```diff
@@ -145,22 +145,26 @@ func (c *controller) brokerDelete(obj interface{}) {
 	glog.V(4).Infof("Received delete event for Broker %v", broker.Name)
 }
 
+// the Message strings have a terminating period and space so they can
+// be easily combined with a follow on specific message.
```

Comment:
we should probably follow whatever pattern kube has.  Although, from my quick scan of a few files I didn't see a consistent pattern so unless someone else can provide us with a concrete design decision from kube we're probably ok with what you did.